src/index.ts

```diff
@@ -49,5 +51,4 @@ const preload = () => preloadAPI;
 export {
     register,
     preload,
-    isCached
```

Comment:
It was a part of public API 😢

Reply:
@IvanIhnatsiuk may I kindly ask you to introduce as small as possible changes?
I want to have in this PR as small as possible changes, and re-organizing exports it's not something that I'd like to do in this PR.
Can you keep this line unchanged, please?